pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/signed-exchange.md

```diff
@@ -22,11 +22,11 @@ Signed Exchange AMP content is delivered _in addition to_ (rather than instead o
 
 To implement Signed Exchanges, you must meet the following requirements:
 
-*   Your site must be hosted on your own HTTPS server and not from a shared host. 
+*   Fairly low-level control over your HTTP server, such as the ability to generate HTTP headers. (Most purely web-based hosting solutions such as Blogger are _not_ be compatible with Signed Exchanges.) 
```

Comment:
low-level control sounds scary (everyone has a different understanding about what low-level means). Can you make this more precise?

Reply:
"are not be compatible" doesn't sound right. My guess is you want to drop "be".

Reply:
thx, updated.